pagopa/io-app
# chore(IT Wallet): [SIW-1441] Add Disability Card to credentials catalog

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/6074

## Short description
This PR adds the Disability Card to the credential catalog

## List of changes proposed in this pull request
- Refactored `WalletCardOnboardingScreen`:
  - moved cards onboarding to `OtherCardsOnboardingSection` component
  - moved ITW credential onboarding to `ItwCredentialOnboardingSection` component
- removed mocked Welfare cards onboarding button
- updated test

## How to test
From the wallet's landing, click on the `Add to wallet` CTA and check if:
- without a valid ITW instance or an eID, you can't see ITW credentials onboarding and only normal cards are visible
- with a valid ITW instance and an eID, you can see both ITW credentials onboarding and normal cards
- if a credential is already present in your wallet, it's module in catalog should display a banner and you shouldn't be able to interact with it

## Preview

<img src="https://github.com/user-attachments/assets/b0e1c369-d6db-4f4e-948f-af5414ea3efd" width="250" />
